src/doc/using_visit/VisualizationWindows/Interactor_settings.rst

```diff
@@ -21,47 +18,27 @@ that have interactor properties. The **Interactors window** is shown in
 Zoom interactor settings
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-The zoom interactor settings are mostly used when the vis window is in
-zoom mode. When the vis window is in zoom mode, clicking in the vis window
-will anchor a point that becomes one of the corners of a zoom rectangle.
-When you release the mouse, the point over which the mouse was released
-becomes the opposite corner of the zoom rectangle. VisIt's default behavior
-is to show guidelines that extend from the edges of the zoom rectangle to
-the edges of the plots' bounding box when the vis window is in 2D mode. If
-you want to turn off the guidelines, click off the **Show Guidelines** check
-box in the **Interactors window**.
+The zoom interactor settings are mostly used when the vis window is in zoom mode. 
+When the vis window is in zoom mode, clicking in the vis window will anchor a point that becomes one of the corners of a zoom rectangle.
+When you release the mouse, the point over which the mouse was released becomes the opposite corner of the zoom rectangle. 
+VisIt's default behavior is to show guidelines that extend from the edges of the zoom rectangle to the edges of the plots' bounding box when the vis window is in 2D mode. 
+If you want to turn off the guidelines, click off the **Show Guidelines** check box in the **Interactors window**.
 
-When sweeping out a zoom rectangle in zoom mode, VisIt allows you to draw
-a rectangle of any proportion. The relative shape of the zoom rectangle,
-in turn, influences the shape of the viewport drawn in the vis window. If
-you hold down the *Shift* key while sweeping out the zoom rectangle, VisIt
-will constrain the shape of the zoom rectangle to a square. If you want
-VisIt to always force a square zoom rectangle so that you don't have to
-use the *Shift* key, you can click on the **Clamp to Square** check box,
-click **Apply** in the **Interactors window** and save your settings.
+When sweeping out a zoom rectangle in zoom mode, VisIt allows you to draw a rectangle of any proportion. 
```

Comment:
Explaining this behavior of the aspect ratio of the zoom box is great to include in our docs.

I am not seeing this behavior on macOS though. Here is what I do see (with a 2D dataset, `curve2d.silo`)...

- If I do depress `Shift` before dragging while in `Zoom` mode, I get panning...which is same as in `Navigate` mode.
- If I depress `Shift` *after* beginning a zoom drag, I get no effect (no constraining)
- I tried other keys as well (`Ctrl`, `Alt`, and `Command` (or the Apple option key). For `Command` I get an inverted zoom event (e.g. it zooms out not in ;)
- I though I recall seeing a constrained drag zoom on macOS in the past but am now unable to do it.